### PR TITLE
Show alert message when user's email is unverified

### LINKED ISSUE #1151: Show an alert when the user email is unverified.
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

**Expected behavior:**

image

### PR DESCRIPTION
**Pull request by pradeepgangwar**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Show alert message on certain routes as per v1 to users whose email is unverified.

#### Changes proposed in this pull request:
- Added message alert box on certain routes.

Fixes #1151

### CODE REVIEW

**Comment by pradeepgangwar** [comment]
This looks something like this.
screenshot from 2018-06-01 02-15-37

Also I am not sure if resend mail functionality works. Please let me know if that works and where i can find that.

File: app/components/unverified-user-message.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import Component from '@ember/component';
+import $ from 'jquery';
+
+export default Component.extend({
+
+  didInsertElement() {
+    $('.message .close')
```
Comment (niranjan94) [inline app/components/unverified-user-message.js]: Don't use jquery. Just hide it with ember.

File: app/templates/components/unverified-user-message.hbs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{{#if session.isAuthenticated}}
+  {{#unless authManager.currentUser.isVerified}}
+    <div class="ui icon error message eight wide column center aligned">
+      <i class="close icon"></i>
+      <i class="info icon"></i>
+      <div class="content">
+        <div class="header">
+          {{#if (includes session.currentRouteName 'events.view.index')}}
```
Comment (niranjan94) [inline app/templates/components/unverified-user-message.hbs]: You can use eq instead of includes

File: app/templates/events/view.hbs [hunk 3]
```diff
@@ -1,5 +1,16 @@
+{{#if (or (includes session.currentRouteName 'events.view.index') (includes session.currentRouteName 'events.view.export'))}}
```
Comment (niranjan94) [inline app/templates/events/view.hbs]: Eq instead of includes please

File: app/components/unverified-user-message.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+
+export default Component.extend({
+
+  isMessageVisible: true,
+
+  showMessage: computed('session.isAuthenticated', 'authManager.currentUser.isVerified', 'isMessageVisible', function() {
+    let isAuthenticated = this.get('session.isAuthenticated');
```
Comment (niranjan94) [inline app/components/unverified-user-message.js]: Why the unnecessary variables and the redundant conditional. ?

You could just do the below right ?

```js
return this.get('isMessageVisible') 
    && this.get('session.isAuthenticated') 
    && !this.get('authManager.currentUser.isVerified');
```

File: app/components/unverified-user-message.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+
+export default Component.extend({
+
+  isMessageVisible: true,
+
+  showMessage: computed('session.isAuthenticated', 'authManager.currentUser.isVerified', 'isMessageVisible', function() {
```
Comment (niranjan94) [inline app/components/unverified-user-message.js]: Call it `shouldShowMessage`

**Comment by pradeepgangwar** [comment]
@niranjan94 Did the changes. Please review.

**Pull request merged**